opm/polymer/fullyimplicit/SimulatorFullyImplicitBlackoilPolymer_impl.hpp

```diff
@@ -110,7 +110,7 @@ namespace Opm
             if (wells_manager.c_wells() == 0) {
                 OPM_THROW(std::runtime_error, "Cannot control polymer injection via WPOLYMER without wells.");
             }
-            polymer_inflow_ptr.reset(new PolymerInflowFromDeck(deck_, BaseType::eclipse_state_, *wells, Opm::UgGridHelpers::numCells(BaseType::grid_), timer.currentStepNum()));
+            polymer_inflow_ptr.reset(new PolymerInflowFromDeck(BaseType::eclipse_state_, *wells, Opm::UgGridHelpers::numCells(BaseType::grid_), timer.currentStepNum()));
```

Comment:
Is there a way to ask the eclipse state if there are polymer wells? If so we could get rid of the deck_ member in this class (now just used on line 109). If there is no such thing, do not bother, it's not a very big deal.